### ISSUE TITLE
npm package lacks files from git

### ISSUE THREAD

**Issue by royappa** [urish/angular2-moment#130]
**Description of the Issue and Steps to Reproduce:**

**Did you search for duplicate issue? [Yes / No]**
Yes

**Please describe the issue and steps to reproduce, preferably with a code sample / plunker:**

I wanted to use the add/subtract functions but they are not available via npm even though the version number and package.json seems to indicate they should be included. Evidence below from my installation. I don't know if the package is incomplete or damaged in any other way, just that these files are missing.

```
$ npm install --save angular2-moment
$ cd node_modules/angular2-moment
$ egrep 'add|subtract' package.json
    "add.pipe.js",
    "add.pipe.js.map",
    "add.pipe.d.ts",
    "add.pipe.metadata.json",
    "subtract.pipe.js",
    "subtract.pipe.js.map",
    "subtract.pipe.d.ts",
    "subtract.pipe.metadata.json",
$ ls -als add* subtract*
ls: cannot access add*: No such file or directory
ls: cannot access subtract*: No such file or directory
$ fgrep version package.json
  "version": "1.3.0"
```
*Ensure your issue is isolated to angular2-moment. Issues involving third party tools will be closed unless submitted by the tool's author/maintainer.*

**Environment:**

Please answer the following questions:

* Angular version? 
* TypeScript version?
* `moment` version?
* Are you using `moment-timezone`? 
* Are you using the angular-cli? 
* Using Rollup/Webpack/System.js/Ionic/similar?

**Comment by urish** [comment]
Fixed, released as 1.3.1

**Comment by royappa** [comment]
Verified fixed. Thank you!
